doc/api/ticker_api.rst

```diff
@@ -6,3 +6,7 @@
    :members:
    :undoc-members:
    :show-inheritance:
+
+
+ .. inheritance-diagram:: matplotlib.ticker
```

Comment:
I think that this needs to a list of the classes you want included in the diagram, not a module.

Reply:
Hmm, it works locally (and specifying the module works elsewhere in the docs), not sure what's going on here...

Reply:
The build log contains this:

> WARNING: invalid signature for automodule ('matplotlib.ticker\n  :members:\n  :undoc-members:\n  :show-inheritance:')
WARNING: don't know which module to import for autodocumenting 'matplotlib.ticker\n  :members:\n  :undoc-members:\n  :show-inheritance:' (try placing a "module" or "currentmodule" directive in the document, or giving an explicit module name)